jsonrpc.go

```diff
@@ -293,6 +299,28 @@ type SSHKeyState struct {
 	SSHKey string `json:"sshKey"`
 }
 
+func rpcGetNameConfig() (NameConfig, error) {
+	return config.NameConfig, nil
+}
+
+func rpcSetNameConfig(deviceName string) (NameConfig, error) {
+	config.NameConfig = NameConfig{
+		Name: deviceName,
+		DNS:  slug.Make(deviceName) + ".local",
```

Comment:
Would be nicer to _not_ force the DNS name to always be computed as `.local` domain... could we do that if they don't give a DNS name (to default to a reasonable value), but honor whatever they typed.. then allow that text to be entered on the UI.